pkgs/by-name/dy/dysk/package.nix

```
@@ -38,5 +38,10 @@ rustPlatform.buildRustPackage rec {
       koral
     ];
     mainProgram = "dysk";
+    platforms = [
+      "x86_64-linux"
+      "aarch64-linux"
+      "i686-linux"
+    ];
```

How about using `lib.platforms.linux`?

```suggestion
    platforms = platforms.linux;
```

---

ok